project/src/components/app/app.tsx

```diff
@@ -3,15 +3,17 @@ import NotFoundView from '../not-found-view/not-found-view';
 import { AppRoutes } from '../../routes';
 import PrivateRoute from '../private-route/private-route';
 import { AuthorizationStatus } from '../../const';
+import { Films } from '../../types/films';
 
 type AppProps = {
   filmGenre: string;
   filmName: string;
   filmReleaseYear: string;
+  films: Films;
 }
 
 function App(
-  { filmGenre, filmName, filmReleaseYear }: AppProps,
+  { filmGenre, filmName, filmReleaseYear, films }: AppProps,
```

Comment:
лучше писать вот так, оно становится читабельней 
```suggestion
  { 
    genre, 
    name,
    releaseDate, 
    films,
  }: AppProps,
```